psycopg/psycopg/pq/_pq_ctypes.pyi

```diff
@@ -175,6 +175,8 @@ def PQexec(arg1: PGconn_struct | None, arg2: bytes) -> PGresult_struct: ...
 def PQexecParams(arg1: PGconn_struct | None, arg2: bytes, arg3: int, arg4: _Pointer[c_uint], arg5: _Pointer[c_char_p], arg6: _Pointer[c_int], arg7: _Pointer[c_int], arg8: int) -> PGresult_struct: ...
 def PQdescribePrepared(arg1: PGconn_struct | None, arg2: bytes) -> PGresult_struct: ...
 def PQdescribePortal(arg1: PGconn_struct | None, arg2: bytes) -> PGresult_struct: ...
+def PQclosePrepared(arg1: PGconn_struct | None, arg2: bytes) -> PGresult_struct: ...
+def PQclosePortal(arg1: PGconn_struct | None, arg2: bytes) -> PGresult_struct: ...
```

Comment:
These additions (and the ones below in that file) seem to be the result of respective lines having been dropped in the previous commit (by mistake, I suspect).

Reply:
Since everything is kept in place at the end, it might not be worth fixing the git history; so feel free to ignore my comment!